cli/src/pcluster/resources/imagebuilder/parallelcluster_validate.yaml

```diff
@@ -0,0 +1,454 @@
+                OS='ubuntu2004'
+              else
+                echo "Operating System '${!RELEASE}' is not supported. Failing build."
+                exit {{ validate.Fail.outputs.stdout }}
+              fi
+
+              echo ${!OS}
+
+      # Get input base AMI Architecture
+      - name: OperatingSystemArchitecture
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              ARCH=$(uname -m)
+              case ${!ARCH} in
+                'x86_64')
+                  echo 'x86_64'
+                  ;;
+                'aarch64')
+                  echo 'arm64'
+                  ;;
+                *)
+                  echo "The '${!ARCH}' architecture is not supported. Failing build."
+                  exit {{ validate.Fail.outputs.stdout }}
+                  ;;
+              esac
+
+      # Get platform name
+      - name: PlatformName
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              OS='{{ validate.OperatingSystemName.outputs.stdout }}'
+
+              if [ `echo "${!OS}" | grep -E '^(alinux|centos)'` ]; then
+                PLATFORM='RHEL'
+              elif [ `echo "${!OS}" | grep -E '^ubuntu'` ]; then
+                PLATFORM='DEBIAN'
+              fi
+
+              echo ${!PLATFORM}
+
+      ### condition ###
+      - name: IntelMPISupported
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              if [ {{ validate.OperatingSystemArchitecture.outputs.stdout }} != 'arm64' ]; then
+                echo "true"
+              else
+                echo "false"
+              fi
+
+      - name: ArmPLSupported
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              if [ {{ validate.OperatingSystemArchitecture.outputs.stdout }} == 'arm64' ]; then
+                echo "true"
+              else
+                echo "false"
+              fi
+
+      - name: LustreSupported
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              ARCHITECTURE='{{ validate.OperatingSystemArchitecture.outputs.stdout }}'
+              OS='{{ validate.OperatingSystemName.outputs.stdout }}'
+              if [ ${!ARCHITECTURE} == 'arm64' ] && [[ ${!OS} =~ ^(ubuntu(18|20)04|centos8|alinux2)$ ]] || [ ${!ARCHITECTURE} == 'x86_64' ]; then
+                echo "true"
+              else
+                echo "false"
+              fi
+
+      ### version ###
+      - name: PythonVersion
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "3.7.10"
+
+      - name: NvidiaDriverVersion
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "450.80.02"
```

Comment:
all this hardcoded versions are not manageable. We need to retrieve the value from the source, i.e. the installed cookbook in `/etc/chef/cookbooks/aws-parallelcluster/attributes/default.rb`

Reply:
done